IDE/WIN10/user_settings.h

```diff
@@ -119,6 +119,9 @@
         #define WOLFSSL_AES_OFB
         #define FP_MAX_BITS 16384
     #endif /* FIPS v5 */
+    #if defined(HAVE_FIPS_VERSION) && (HAVE_FIPS_VERSION >= 7)
```

Comment:
This should be `HAVE_FIPS_VERSION >= 6`.

Reply:
fixed